.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
Alright, would you prefer to not have a restriction and instead a recommendation? I can do that too.